guides/source/form_helpers.md

```diff
@@ -766,21 +846,16 @@ module ApplicationHelper
 end
 ```
 
+And use it in a form as usual:
+
 ```erb
 <%= form_with model: @person do |form| %>
   <%= text_field_with_label form, :first_name %>
 <% end %>
 ```
 
-can be replaced with
-
-```erb
-<%= form_with model: @person, builder: LabellingFormBuilder do |form| %>
-  <%= form.text_field :first_name %>
-<% end %>
-```
-
-by defining a `LabellingFormBuilder` class similar to the following:
+But you can also create a subclass of `ActionView::Helpers::FormBuilder`, and
+add the helpers there. After defining this `LabellingFormBuilder` subclass:
 
 ```ruby
 class LabellingFormBuilder < ActionView::Helpers::FormBuilder
```

Comment:
The keyword 'super' refers to the superclasss in OOP in general. In this case since we are extending the `ActionView::Helpers::FormBuilder` class, it refers to that class.

Reply:
Added a comment to clarify